### ISSUE TITLE
Error in PERIPHERAL_CLOCK_I2S_ENABLE() for BL70x.

### ISSUE THREAD

**Issue by pvvx** [bouffalolab/bl_mcu_sdk#74]
I2S is not working.

[PERIPHERAL_CLOCK_I2S_ENABLE()](https://github.com/bouffalolab/bl_mcu_sdk/blob/master/drivers/lhal/include/bflb_clock.h#L147)

```
#define PERIPHERAL_CLOCK_I2S_ENABLE()                             \
    do {                                                          \
        volatile uint32_t regval = getreg32(BFLB_GLB_CGEN1_BASE); \
        regval |= (1 << 27);                                      \
        putreg32(regval, BFLB_GLB_CGEN1_BASE);                    \
    } while (0)
```
BL702/704/706 Reference Manual:
3.3.9 cgen_cfg1
Address：0x40000024 
i2s and qdec2 clock ungate enable - bit 26

If corrected, then I2S will work.

**Comment by sakumisu** [comment]
I2S lhal driver only works for bl616 and bl808 now.

**Comment by pvvx** [comment]
I connected PCM1802 for tests and got it working on all CLK variants.

![image](https://user-images.githubusercontent.com/12629515/214537211-eeed4d91-532a-4ae2-947c-142133352cf0.png)

Stereo 96 kHz 24 bit is successfully transmitted from PCM1802 and BL702 via USB.
The PDS_Set_Audio_PLL_Freq() function has only 5 fixed frequencies, which is inconvenient.
For the project, I needed to change the MCLK frequency, but there is no documentation on setting up the AUDIO PLL.
I had to make research to generate any frequency from 0.5 to 80 MHz with a small step to output from the chip (GLB_Set_I2S_CLK(ENABLE, GLB_I2S_OUT_REF_CLK_SRC)) ;
